src/go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/Diarkis/diarkis-server-template
 go 1.22
 
 require github.com/Diarkis/diarkis v1.1.0-alpha6
```

Comment:
@hobein 
Can you also bump up to `v1.1.0-alpha7`?

Reply:
@nakakoh done